docs/package.json

```diff
@@ -103,7 +103,7 @@
     "react-router": "^5.0.0",
     "react-router-dom": "^5.0.1",
     "react-spring": "^8.0.27",
-    "react-swipeable-views": "^0.13.9",
+    "react-swipeable-views": "^0.14.0-alpha.0",
```

Comment:
What's this change about?

Reply:
it's about these warnings: https://github.com/oliviertassinari/react-swipeable-views/pull/616/files#diff-553671f8b219ec98c93073d5f6d5740aee7cedd4a78a429f51776807216841e4R110